provider/pkg/schema/metadata.go

```diff
@@ -20,6 +20,7 @@ type CloudAPIResource struct {
 	Required          []string                        `json:"required,omitempty"`
 	CreateOnly        []string                        `json:"createOnly,omitempty"`
 	WriteOnly         []string                        `json:"writeOnly,omitempty"`
+	UnorderedArrays   []string                        `json:"unorderedArrays,omitempty"`
```

Comment:
I didn't see this used in here; I assume it's logically part of the follow up review?

Reply:
Yes, was going to be in the follow-up though I think I overlooked this in the next PR 👀

Reply:
Have taken this out for now. This will be for customising diff correctly when they're arrays.